<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuntherWindow</class>
 <widget class="QMainWindow" name="GuntherWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>G체nther</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="GuntherTabs">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>640</width>
      <height>480</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>640</width>
      <height>480</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>640</width>
      <height>480</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Jura</family>
     </font>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tab_data">
     <attribute name="title">
      <string>Data</string>
     </attribute>
     <widget class="QGroupBox" name="GroupData">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>10</y>
        <width>351</width>
        <height>431</height>
       </rect>
      </property>
      <property name="title">
       <string>Datos de la transmisi처n:</string>
      </property>
      <widget class="QLabel" name="LabelNombreTransmision">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>361</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>Nombre: 
</string>
       </property>
      </widget>
      <widget class="QLabel" name="LabelDescripcionTransmision">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>100</y>
         <width>361</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>Descripci처n:
</string>
       </property>
      </widget>
     </widget>
     <widget class="QTreeWidget" name="TreeData">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>256</width>
        <height>431</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Jura</family>
       </font>
      </property>
      <property name="animated">
       <bool>false</bool>
      </property>
      <attribute name="headerVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_inputs">
     <attribute name="title">
      <string>Inputs</string>
     </attribute>
     <widget class="QScrollArea" name="ScrollAreaInputs">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>660</width>
        <height>441</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="widgetResizable">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <widget class="QWidget" name="contenidoInputs">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>639</width>
         <height>399</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>639</width>
         <height>399</height>
        </size>
       </property>
       <widget class="QWidget" name="horizontalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>631</width>
          <height>391</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="LayoutInputs">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinimumSize</enum>
         </property>
        </layout>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_outputs">
     <attribute name="title">
      <string>Outputs</string>
     </attribute>
     <widget class="QScrollArea" name="ScrollAreaOutputs">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>660</width>
        <height>430</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QWidget" name="contenidoOutputs">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>658</width>
         <height>428</height>
        </rect>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>631</width>
          <height>431</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="LayoutOutputs">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinimumSize</enum>
         </property>
        </layout>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_about">
     <attribute name="title">
      <string>Acerca de...</string>
     </attribute>
     <widget class="QScrollArea" name="ScrollAreaAbout">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>641</width>
        <height>441</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="contenidoAbout">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>639</width>
         <height>439</height>
        </rect>
       </property>
       <widget class="QTextEdit" name="TextAbout">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>150</y>
          <width>451</width>
          <height>281</height>
         </rect>
        </property>
        <property name="locale">
         <locale language="Spanish" country="Argentina"/>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="undoRedoEnabled">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::TextBrowserInteraction</set>
        </property>
       </widget>
       <widget class="QGraphicsView" name="graphicsView">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>641</width>
          <height>111</height>
         </rect>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-image: url(:/general/GuntherLogo.png);
background-position: 0px 0px;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="backgroundBrush">
         <brush brushstyle="NoBrush">
          <color alpha="0">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </property>
        <property name="interactive">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLabel" name="LabelVersion">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>120</y>
          <width>631</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Ubuntu</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Versi처n 0.3.5</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="LabelURL">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>410</y>
          <width>181</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>www.radiocefyl.com.ar</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="ui.qrc"/>
 </resources>
 <connections/>
</ui>
